packages/internal-tooling/eslint/index.js

```diff
@@ -115,6 +115,12 @@ module.exports = {
         '@stardust-ui/no-visibility-modifiers': 'error',
       },
     },
+    {
+      files: '**/*.{ts,tsx}',
+      rules: {
+        'no-dupe-class-members': 'off',
```

Comment:
Otherwise we get false positive in `RendererMock.tsx`, see https://github.com/typescript-eslint/typescript-eslint/issues/291

Reply:
No longer needed at the moment, but I think it will be useful to keep this change, as described in the link above

Reply:
Yep, let's keep it